java/quarkus/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>benchmarker.the</groupId>
+  <artifactId>quarkus</artifactId>
+  <version>1.0.0</version>
+  <properties>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <maven.compiler.parameters>true</maven.compiler.parameters>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+
+    <quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
+    <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
+    <quarkus.platform.version>[1.11,1.12)</quarkus.platform.version>
```

Comment:
This doesn't work for me for some reasons. I think it has to do with the fact that the version is with a `Final` string at the end.

Reply:
any idea why @geoand ?
https://github.com/the-benchmarker/web-frameworks/pull/3813/checks?check_run_id=1754048979#step:6:7772

Reply:
Not really. We've never really tested Maven ranges

Reply:
That's what I thought. I've never seen them used (but @waghanza likes it so much 😛 )